suite/alerts/alerts_test.go

```diff
@@ -1,54 +1,24 @@
 package alerts
 
 import (
-	"fmt"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/smartcontractkit/integrations-framework/actions"
-	"github.com/smartcontractkit/integrations-framework/client"
 	"github.com/smartcontractkit/integrations-framework/environment"
-	"github.com/smartcontractkit/integrations-framework/tools"
+	"github.com/smartcontractkit/integrations-framework/suite/testcommon"
 )
 
-var _ = Describe("Alerts suite", func() {
-	var (
-		suiteSetup     *actions.DefaultSuiteSetup
-		adapter        environment.ExternalAdapter
-		chainlinkNodes []client.Chainlink
-		explorer       *client.ExplorerClient
-		err            error
-	)
-
-	BeforeEach(func() {
-		By("Deploying the environment", func() {
-			suiteSetup, err = actions.DefaultLocalSetup(
-				environment.NewChainlinkClusterForAlertsTesting(3),
-				client.NewNetworkFromConfig,
-				tools.ProjectRoot,
-			)
-			Expect(err).ShouldNot(HaveOccurred())
-
-			explorer, err = environment.GetExplorerClientFromEnv(suiteSetup.Env)
-			Expect(err).ShouldNot(HaveOccurred())
-			fmt.Println(explorer.BaseURL)
-
-			chainlinkNodes, err = environment.GetChainlinkClients(suiteSetup.Env)
-			Expect(err).ShouldNot(HaveOccurred())
-			fmt.Println(chainlinkNodes[0].URL())
-
-			adapter, err = environment.GetExternalAdapter(suiteSetup.Env)
-			Expect(err).ShouldNot(HaveOccurred())
-			fmt.Println(adapter.ClusterURL())
-		})
-	})
-
+var _ = FDescribe("Alerts suite", func() {
 	Describe("Alerts", func() {
 		It("Test 1", func() {
```

Comment:
Can we have more explicit naming for that test? Is it testing just the deployment with OPTE?